moduleframework/common.py

```diff
@@ -406,21 +417,11 @@ def getPackageList(self, profile=None):
         package_list = []
         if not profile:
             if 'packages' in self.config:
-                packages_rpm = self.config['packages'].get('rpms') if self.config[
-                    'packages'].get('rpms') else []
+                packages_rpm = self.config.get('packages',{}).get('rpms', [])
```

Comment:
if 'packages' are empty, then it will traceback on 'rpms', I think. This should be catched by an exception like KeyError. What do you think?

Reply:
no, it works well, second param for get is default value instead of None, so that next .get will not traceback